test/check-application

```diff
@@ -93,9 +93,26 @@ class TestApplication(testlib.MachineCase):
         self.filter_containers("all")
         b.wait_present('#containers-containers tr:contains("alpine:latest")')
         b.click('#containers-containers tbody tr:contains("alpine:latest") td.listing-ct-toggle')
+        # open commit modal and close it using cancel
         b.wait_present('#containers-containers tbody tr:contains("alpine:latest") + tr button.btn-commit')
         b.click('#containers-containers tbody tr:contains("alpine:latest") + tr button.btn-commit')
         self.confirm_modal("btn-ctr-cancel-commit")
+        # open commit modal and create an image
+        b.click('#containers-containers tbody tr:contains("alpine:latest") + tr button.btn-commit')
+        b.wait_present(".modal-dialog div")
+        # set commit parameters
+        b.set_input_text("#imageName", "testimg")
+        b.set_input_text("#tag", "testtag")
+        b.set_input_text("#author", "tester")
+        b.set_input_text("#command", "sleep 6000")
+        b.click(".modal-dialog div .btn-ctr-commit")
+        b.wait_not_present(".modal-dialog div")
+        b.wait_present('#containers-images tr:contains("testimg:testtag")')
+        b.click('#containers-images tbody tr:contains("testimg:testtag") td.listing-ct-toggle')
+        # open the listing toggle of testimg and check the commit paramerters
+        b.wait_present('#containers-images tbody tr:contains("testimg:testtag"):has(dd:contains("localhost/testimg:testtag"))')
+        b.wait_present('#containers-images tbody tr:contains("testimg:testtag"):has(dd:contains("sleep 6000"))')
+        b.wait_present('#containers-images tbody tr:contains("testimg:testtag"):has(dd:contains("tester"))')
```

Comment:
There should be also test for failed image creation from commited container.
Notification in the UI should be checked in this case.